### PR TITLE
Kw-feature mycampaigns

### PR DESCRIPTION
**Pull request by KanikaWadhawan**
# Add MyCampaigns Screen Components

- Made three different view of mycampaigns screen for all the different scenarios 
- Style the three screens 
- create two components for button types
- Add italic font in Theme
- modify .gitignore

### CODE REVIEW

File: client/js/components/SmallSubscribeButton/SmallSubscribeButton.js [hunk 1]
```diff
@@ -1,2 +1,13 @@
 import React, {Component} from 'react';
 import styles from './styles';
+import {TouchableOpacity, Text} from 'react-native';
+
+const SmallSubscribeButton = props => {
+  return (
+    <TouchableOpacity style={styles.button} onPress={() => {}}>
```
Comment (iyuvraajsingh) [inline client/js/components/SmallSubscribeButton/SmallSubscribeButton.js]: Where this onPress is leading to?

File: client/js/components/SmallSubscribeButton/styles.js [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+} = THEME;
+const styles = StyleSheet.create({
+  button: {
+    borderColor: white,
+    borderWidth: 1,
+    borderRadius: baseSpacing * 2,
+    padding: baseSpacing,
+    padding: baseSpacing,
+    height: 50,
+    width: 104, //Width 100% does not work
```
Comment (iyuvraajsingh) [inline client/js/components/SmallSubscribeButton/styles.js]: Remove the comment.

File: client/js/components/SmallUnsubscribeButton/SmallUnsubscribeButton.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React, {Component} from 'react';
+import styles from './styles';
+import {TouchableOpacity, Text} from 'react-native';
+
+const SmallUnsubscribeButton = props => {
+  return (
+    <TouchableOpacity style={styles.button} onPress={() => {}}>
```
Comment (iyuvraajsingh) [inline client/js/components/SmallUnsubscribeButton/SmallUnsubscribeButton.js]: Where this onPress is leading to?

File: client/js/components/SmallUnsubscribeButton/styles.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+} = THEME;
+const styles = StyleSheet.create({
+  button: {
+    borderColor: white,
+    borderWidth: 1,
+    borderRadius: baseSpacing * 2,
+    padding: baseSpacing,
+    padding: baseSpacing,
+    height: 50,
+    width: 104, //Width 100% does not work
```
Comment (iyuvraajsingh) [inline client/js/components/SmallUnsubscribeButton/styles.js]: Remove the comment.

File: client/js/screens/MyCampaigns/MyCampaigns.js [hunk 5]
```diff
@@ -1,8 +1,135 @@
...
+          <View style={styles.buttonHolder}>
+            <TouchableOpacity style={styles.button} onPress={() => {}}>
+              <ImageBackground source={ActiveButton} style={styles.buttonImage}>
+                <Text style={styles.buttonText}>Sign In</Text>
+              </ImageBackground>
+            </TouchableOpacity>
+          </View>
+          <Text style={{textAlign: 'center'}}>Or</Text>
+          <View style={styles.buttonHolder}>
+            <TouchableOpacity style={styles.button} onPress={() => {}}>
```
Comment (iyuvraajsingh) [inline client/js/screens/MyCampaigns/MyCampaigns.js]: Where this onPress is leading to?

File: client/js/screens/MyCampaigns/MyCampaigns.js [hunk 6]
```diff
@@ -1,8 +1,135 @@
...
+          </View>
+          <Text style={styles.pageTitle}>
+            Sign in to keep track of campaigns and events
+          </Text>
+          <Text style={styles.noCampaign}>
+            Keep your calendar updated and never miss out to be a part of the
+            movement!
+          </Text>
+          <View style={styles.buttonHolder}>
+            <TouchableOpacity style={styles.button} onPress={() => {}}>
```
Comment (iyuvraajsingh) [inline client/js/screens/MyCampaigns/MyCampaigns.js]: Where this onPress is leading to?

File: client/js/screens/MyCampaigns/MyCampaigns.js [hunk 7]
```diff
@@ -1,8 +1,135 @@
...
+            You are not following any campaigns yet
+          </Text>
+
+          <Text style={styles.noCampaign}>
+            Get the wheel rolling and ignite the fire by following campaigns and
+            adding events to your stuff. Keep your calendar updated and never
+            miss out to be a part of the movement.
+          </Text>
+          <View style={styles.buttonHolder}>
+            <TouchableOpacity style={styles.button} onPress={() => {}}>
```
Comment (iyuvraajsingh) [inline client/js/screens/MyCampaigns/MyCampaigns.js]: Where this onPress is leading to?

**Review by iyuvraajsingh — CHANGES_REQUESTED** [review]

File: client/js/screens/MyCampaigns/MyCampaigns.js (hunk 5 above)
Comment (KanikaWadhawan) [reply client/js/screens/MyCampaigns/MyCampaigns.js]: Done

File: client/js/screens/MyCampaigns/MyCampaigns.js (hunk 6 above)
Comment (KanikaWadhawan) [reply client/js/screens/MyCampaigns/MyCampaigns.js]: Done

File: client/js/screens/MyCampaigns/MyCampaigns.js (hunk 7 above)
Comment (KanikaWadhawan) [reply client/js/screens/MyCampaigns/MyCampaigns.js]: Done

File: client/js/components/SmallUnsubscribeButton/styles.js (hunk 4 above)
Comment (KanikaWadhawan) [reply client/js/components/SmallUnsubscribeButton/styles.js]: Done

File: client/js/components/SmallUnsubscribeButton/SmallUnsubscribeButton.js (hunk 3 above)
Comment (KanikaWadhawan) [reply client/js/components/SmallUnsubscribeButton/SmallUnsubscribeButton.js]: Done

File: client/js/components/SmallSubscribeButton/styles.js (hunk 2 above)
Comment (KanikaWadhawan) [reply client/js/components/SmallSubscribeButton/styles.js]: Done

File: client/js/components/SmallSubscribeButton/SmallSubscribeButton.js (hunk 1 above)
Comment (KanikaWadhawan) [reply client/js/components/SmallSubscribeButton/SmallSubscribeButton.js]: Done

File: client/js/components/SmallUnsubscribeButton/SmallUnsubscribeButton.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+import {TouchableOpacity, Text} from 'react-native';
+
+const SmallUnsubscribeButton = props => {
+  return (
+    <TouchableOpacity style={styles.button}>
+      <Text style={styles.buttonText}>Unsubscribe</Text>
+    </TouchableOpacity>
+  );
+};
+
```
Comment (siddharthparmar7) [inline client/js/components/SmallUnsubscribeButton/SmallUnsubscribeButton.js:12]: Can you create a button that can toggle between subscribe and unsubscribe?

File: client/js/components/SmallUnsubscribeButton/styles.js [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+} = THEME;
+const styles = StyleSheet.create({
+  button: {
+    borderColor: white,
+    borderWidth: 1,
+    borderRadius: baseSpacing * 2,
+    padding: baseSpacing,
+    padding: baseSpacing,
+    height: 50,
+    width: 104,
```
Comment (siddharthparmar7) [inline client/js/components/SmallUnsubscribeButton/styles.js]: use %s

File: client/js/screens/MyCampaigns/MyCampaigns.js [hunk 10]
```diff
@@ -1,8 +1,135 @@
...
+  return (
+    <View style={styles.item}>
+      <View>
+        <Text style={styles.title}>{title}</Text>
+        <Text style={styles.category}>{category}</Text>
+      </View>
+      <SmallUnsubscribeButton />
+    </View>
+  );
+}
```
Comment (siddharthparmar7) [inline client/js/screens/MyCampaigns/MyCampaigns.js]: Please convert in to an arrow function?

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work. Please make the requested changes.

File: client/js/components/SmallUnsubscribeButton/SmallUnsubscribeButton.js (hunk 8 above)
Comment (KanikaWadhawan) [reply client/js/components/SmallUnsubscribeButton/SmallUnsubscribeButton.js:12]: Hey Sid, that’s not a part of the Design Requirements.

File: client/js/components/SmallUnsubscribeButton/styles.js (hunk 9 above)
Comment (KanikaWadhawan) [reply client/js/components/SmallUnsubscribeButton/styles.js]: Sure

File: client/js/screens/MyCampaigns/MyCampaigns.js (hunk 10 above)
Comment (KanikaWadhawan) [reply client/js/screens/MyCampaigns/MyCampaigns.js]: Sure

File: client/js/components/SmallUnsubscribeButton/SmallUnsubscribeButton.js (hunk 8 above)
Comment (siddharthparmar7) [reply client/js/components/SmallUnsubscribeButton/SmallUnsubscribeButton.js:12]: What do you mean? slack me.

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**